core/services/webhook/validate_test.go

```diff
@@ -78,53 +75,69 @@ func TestValidatedWebJobSpec(t *testing.T) {
                 ds -> ds_parse -> ds_multiply;
             """
             `,
-			findError: nil,
 			assertion: func(t *testing.T, s job.Job, err error) {
-				require.Error(t, err)
-				require.Equal(t, webhook.ErrMissingJobID, errors.Cause(err))
+				require.NoError(t, err)
 			},
 		},
 		{
-			name: "EI does not exist",
+			name: "with multiple external initiators and externalJobID",
 			toml: `
             type            = "webhook"
             schemaVersion   = 1
-            externalJobID           = "0EEC7E1D-D0D2-476C-A1A8-72DFB6633F46"
-            externalInitiatorName = "foo"
-            externalInitiatorSpec = "foo"
+            externalJobID   = "0eec7e1d-d0d2-476c-a1a8-72dfb6633f46"
+			externalInitiators = [
+				{ name = "foo", spec = '{"foo": 42}' },
```

Comment:
Could we just put the name inside spec to avoid the nested json? i.e.
```
externalInitiators = [{"name":"foo", "foo":42}, {"name":"bar", "bar":42}]
```

Reply:
This won't work, since it's not JSON - the outer wrapping is TOML